packages/web-components/src/utils/table.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx React.createElement */
+import React from 'react';
```

Comment:
The `@jsx` comment is [changing typescript's `jsxFactory`](https://www.typescriptlang.org/tsconfig#jsxFactory) for this specific file, and the React import is so I can use that factory instead of `import { h } from `@stencil/core;`